website/snippets/_new-sl-setup.md

```diff
@@ -1,31 +1,54 @@
 
-4. In the **Set Up Semantic Layer Configuration** page, enter the credentials you want the Semantic Layer to use specific to your data platform.
+- In the **Semantic Layer & Credentials** page, select the deployment environment you want for the Semantic Layer and click **Save**. This provides administrators with the flexibility to choose the environment where the Semantic Layer will be enabled. <br />
+
+### 2. Add Semantic Layer credential and service token
+- <Expandable alt_header="Learn more about granular access control">
+
+    You can authenticate to the dbt Semantic Layer in downstream interfaces using a [service token](/docs/dbt-cloud-apis/service-tokens) for more granular control. The credential configuration allows admins to map service tokens to the appropriate credentials, which can then be distributed to the relevant teams for BI connection setup.
+
+    - Admins can link multiple service tokens to a one single credential within a project.  However, each service token can only be linked to one credential per project.
+    - When you send a request through our APIs, the service token you provide will be linked to the corresponding credential and will follow access policies of the data used to build your semantic objects.
+    - [Environment variables](/docs/build/environment-variables), like `{{env_var('DBT_WAREHOUSE')}` aren't supported the dbt Semantic Layer yet. You must use the actual credentials instead.
+
+ </Expandable>
 
-     - Use credentials with minimal privileges. This is because the Semantic Layer requires read access to the schema(s) containing the dbt models used in your semantic models for downstream applications
-     - Note, [Environment variables](/docs/build/environment-variables) such as `{{env_var('DBT_WAREHOUSE')}`, doesn't supported the dbt Semantic Layer yet. You must use the actual credentials.
+- On the **Credentials & service tokens** page, click the **Add Semantic Layer credential** button to configure the credentials you want the Semantic Layer to use specific to your data platform. You can configure as many credentials as you need for more granular control.
+- In the **Add credentials** section, fill in the data platform's credentials, which will link to a [service tokens](/docs/dbt-cloud-apis/service-tokens). 
+    We recommend using “read-only” credentials. You can also use your deployment credentials, but only for testing purposes.
+- In the **Map new service token** section,  map a service token to the credential you configured in the previous step. 
+- dbt Cloud automatically selects the service token permission set you need (Semantic Layer Only and Metadata Only).
 
-<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-configure-sl.jpg" width="45%" title="Enter the credentials you want the Semantic Layer to use specific to your data platform, and select the deployment environment."/>
+- Click **Save** to link the service token to the credential. 
```

Comment:
we say [here](https://docs.getdbt.com/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) that integrations should use pat tokens -- what's the best route for users: service tokens or pat?

Reply:
not pat - update doc